lix/packages/sdk/src/combine/combine-changes.ts

```diff
@@ -0,0 +1,106 @@
+import type { LixPlugin } from "../plugin.js";
+import type { Change } from "../schema.js";
+import type { Lix } from "../types.js";
+
+/**
+ * Combines the changes of two lixes into the target lix.
+ */
+export async function combineChanges(args: { target: Lix; source: Lix }): Promise<void> {
+	const toBeAppliedChanges: Record<LixPlugin["key"], Array<Change>> = {};
+
+	// TODO ideally a right join via sqlite attaching works
+	//      instead of fetching all changes from source
```

Comment:
what happens if source allready has conflicts?

Reply:
The conflict is ignored as it is tracked already. I made this explicit and added a test https://github.com/opral/monorepo/pull/3067/commits/6aeb7408f7fa8e8c783c0da867765b53b4f6c8f5

Reply:
@samuelstroschein  you cannot just ignore conflicts , this will lead to dataloss. the conflict is tracked in the source, not in the target!